src/main/java/org/wso2/lsp4intellij/client/languageserver/serverdefinition/ServerListener.java

```diff
@@ -0,0 +1,15 @@
+package org.wso2.lsp4intellij.client.languageserver.serverdefinition;
+
+import org.eclipse.lsp4j.InitializeResult;
+import org.eclipse.lsp4j.services.LanguageServer;
+import org.jetbrains.annotations.NotNull;
+
+public interface ServerListener {
+
+  public static final ServerListener NO_OP = new ServerListener() {
```

Comment:
I would love if it is called `INSTANCE`

Reply:
Well this is a empty or noop implementation which is used as default value. Is it correct to call it as INSTANCE ? what about DEFAULT ?

Reply:
`DEFAULT` make sense to me

Reply:
@rasika changed to DEFAULT